resources/assets/lib/beatmapset-panel.tsx

```diff
@@ -415,6 +421,7 @@ export default class BeatmapsetPanel extends React.Component<Props> {
           <a className='beatmapset-panel__main-link u-ellipsis-overflow' href={this.url}>
             {osu.trans('beatmapsets.show.details.by_artist', { artist: getArtist(this.props.beatmapset) })}
           </a>
+          {(this.props.beatmapset.track !== null) && <FeaturedArtistBadge />}
```

Comment:
this check doesn't work because `!== null` is false when `track === undefined` (unless there's specific reason, `!= null` is generally preferred)

Reply:
actually, this won't even work without `track` being included. Use `track_id` instead